### PR TITLE
37 add yield information table

### LINKED ISSUE #37: Add Yield information table

### PR DESCRIPTION
**Pull request by Rashmil-1999**
With this PR I have added the Yeild information for Drought Resistant Seed. Please download the new data.json file from box and run `migrations_forward.sh` to apply new changes.

### CODE REVIEW

File: alembic/versions/20240425_1833_add_drs_yeild_table.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "drought_resistant_seed_yield",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("replicate", sa.Integer(), nullable=True),
+        sa.Column("line", sa.String(length=200), nullable=False),
+        sa.Column("planting_date", sa.Date(), nullable=False),
+        sa.Column("harvest_date", sa.Date(), nullable=False),
+        sa.Column("yield_amount", sa.Float(), nullable=False),
```
Comment (navarroc) [inline alembic/versions/20240425_1833_add_drs_yeild_table.py]: Do we just want to have this be yield?

File: app/api/router.py [hunk 2]
```diff
@@ -11,3 +11,6 @@
 api_router.include_router(farms_router, prefix="/farms", tags=["farms"])
 api_router.include_router(fields_router, prefix="/fields", tags=["fields"])
 api_router.include_router(research_router, prefix="/research", tags=["research"])
+api_router.include_router(
+    drs_yield_router, prefix="/drs", tags=["drought_resistant_seed_yield"]
```
Comment (navarroc) [inline app/api/router.py]: Do we want this to be a little more descriptive? Maybe drought-resistant-seed? I know it's likely we only consume the API, but drs is very non descriptive.

File: alembic/versions/20240425_1833_add_drs_yeild_table.py (hunk 1 above)
Comment (Rashmil-1999) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: yeah that sounds cool to me

File: app/api/router.py (hunk 2 above)
Comment (Rashmil-1999) [reply app/api/router.py]: yeah I thought so but my brain was fried up by thinking names for fields and all the other stuff lol.... I will update it...

Comment (navarroc) [reply app/api/router.py]: No worries, thanks for taking a look.

File: alembic/versions/20240425_1833_add_drs_yeild_table.py (hunk 1 above)
Comment (Rashmil-1999) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: ahh, yield is not allowed as it is a keyword in python :P

Comment (Rashmil-1999) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: do you still want me to change it?

Comment (navarroc) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: Maybe crop_yield?

Comment (Rashmil-1999) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: okay

Comment (Rashmil-1999) [reply alembic/versions/20240425_1833_add_drs_yeild_table.py]: Done...

File: app/api/router.py (hunk 2 above)
Comment (Rashmil-1999) [reply app/api/router.py]: done...

File: app/models/__init__.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from .drs_yeild import DroughtResistantSeedYield
```
Comment (navarroc) [inline app/models/__init__.py]: yield is misspelled.

Comment (Rashmil-1999) [reply app/models/__init__.py]: ohh nooo :(

Comment (Rashmil-1999) [reply app/models/__init__.py]: okay fixed

**Review by navarroc — APPROVED** [review]

**Pull request merged**